ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -157,6 +157,18 @@ if [ ${SECURE_BOOT_ENABLED} = "true" ]; then
   sed -i '/guest_id/a\  firmware         = "efi"\n  efi_secure_boot_enabled = "true"' ./vm/main.tf
 fi
 
+if [ "${STORAGE_POLICY_ID}" != "" ]; then
+  # shellcheck source=/dev/null
+  source "${SHARED_DIR}/govc.sh"
+  govc storage.policy.ls "${STORAGE_POLICY_ID}"
+  ret="$?"
+  if [ $ret -ne 0 ]; then
+    echo "ERROR: Can not find storage policy id ${STORAGE_POLICY_ID}"
+    exit "$ret"
+  fi
+  sed -i "/guest_id/a\ storage_policy_id = \"${STORAGE_POLICY_ID}"\" ./vm/main.tf
```

Comment:
can we use `sed -i "/guest_id/a\ storage_policy_id = '${STORAGE_POLICY_ID}'" ./vm/main.tf` here?

Reply:
that depends on terraform, is single-quote supported? https://developer.hashicorp.com/terraform/language/expressions/strings#quoted-strings

Reply:
okay, then keep the double-quote for safety. But seem like the last `\` in the line is placed in a wrong place.